AdvancedApplications/ODBCApplication/README.md

```diff
@@ -1,20 +1,29 @@
 
-This assumes that you have set up the environment to run a ODBC/DB2CLI application. If you haven't done so, [please follow the instructions here](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/tree/master/AdvancedApplications#odbcdb2cli-setup) to set up your environment. 
+If you are not running in the Docker container, run the script to configure SSL which is used when setting up the container which can be found [here](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/blob/master/container/setup/setup-ssl.sh). The script downloads the SSL server certificate through the REST API. This is necessary as Db2 Event Store is configured with SSL with a default keystore out of the box (this may change if you configure your own keystore after installation). Also the following environment variables must be set up
 
-If not running within the docker container, run the script to configure SSL that is provided in the container [set up folder in this repository](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/blob/master/container/setup/setup-ssl.sh), which will download the SSL server certificate through the rest API. This is necessary because Db2 Event Store is configured with SSL with a default keystore out of the box (this may change if you configure your own keystore after installation). Also set up the environment variables IP with the cluster IP address, EVENT_USER with the user name, and EVENT_PASSWORD with the user password.
+* IP with the cluster IP address
+* EVENT_USER with the user name
+* EVENT_PASSWORD with the user password
 
-### To run the example app [`ExampleODBCApp.c`](ExampleODBCApp.c), follow these steps:
-1. Compile the app by running [`./bldExampleODBCApp.c`](bldExampleODBCApp)`<odbc_client_path>`. The `<odbc_client_path>` is where you have unpacked your ODBC client at the environment setup stage.
-    * The script will then do the following for you:
-       * Find the ODBC headers under `<odbc_client_path>/include` and compile the main app [`ExampleODBCApp.c`](ExampleODBCApp.c) and a helper [`utilcli.c`](utilcli.c) which will check for errors and return the diagnostic messages
-       * Link the db2 library under `<odbc_client_path>/lib`.
-2. Run the executable `./ExampleODBCApp`
+## Running the example app ExampleODBCApp.c
 
-### To clean up the compiler generated files, run:
-[`./bldExampleODBCApp.c`](bldExampleODBCApp) --clean
+1. Compile the app by running
+
+`./bldExampleODBCApp <odbc_client_path>` 
+The `<odbc_client_path>` is where you unpacked the ODBC client during the environment setup. The script does the following:
+     * Finds the ODBC headers under `<odbc_client_path>/include` and compiles the main app ExampleODBCApp.c and a helper utilcli.c which checks for errors and returns diagnostic messages
```

Comment:
there is an issue with the formatting of these bullets, they all come up in the same paragraph

Reply:
moved stuff around and fixed it